config/routes.rb

```diff
@@ -1,11 +1,14 @@
 Rails.application.routes.draw do
   delete 'todos', to: 'todos#destroy'
-  resources :todos, only: [:index, :create, :update, :destroy] do
+  resources :todos, only: [:index, :create, :update] do
     collection do
       delete :delete_all, to: 'todos#delete_all'
-      get :get_owners, to: 'todos#get_owners'
     end
   end
   delete 'users', to: 'users#destroy'
-  resources :users, only: [:index, :create]
+  resources :users, only: [:index, :create] do
+    collection do
+      get :all, to: 'users#get_all_users'
```

Comment:
`get_all_users` petqa ?

`index` -y gidum ta lox casery-in hti bol kini..

Reply:
Ete `get_all_users`-y inivech, es `axios`-av hunc bidi `index`-in haskacnim, ver lox usernerin veradardzne, vech tvyal `page`-in ev `per`-in hamapatasxanox?

Reply:
Prosto haskanumchum, xeya qez All Usernen petq ete Pagination onis.

Lav isk ete petqa apa prosto karis mihat check xarkis, orinak `witohut_pagination` kam `only_all_users` ev ete `true` apa servisich veradardznums lox aranch paginationi.

Reply:
👍